lib/node_modules/@stdlib/assert/deep-has-own-property/test/test.validate.js

```diff
@@ -0,0 +1,102 @@
+'use strict';
+
+// MODULES //
+
+var tape = require( 'tape' );
+var validate = require( './../lib/validate.js' );
+
+
+// TESTS //
+
+tape( 'main export is a function', function test( test ) {
+	test.ok( true, __filename );
+	test.strictEqual( typeof validate, 'function', 'export is a function' );
+	test.end();
+});
+
+tape( 'the function returns an error if provided an options argument which is not an object', function test( test ) {
+	var err;
```

Comment:
Prefer array iteration for reasons of maintenance. If everything is unrolled, as is now, then, to add a new test value, we have to copy 4 lines of code. Better is to simply add another value to a test value list.

Further, suppose we wanted to update the test message. Here, we would have to update the test message for each assertion.

I can understand the motivation for being explicit and we do try to be explicit for certain kinds of tests, but here is not one of those times.